CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## Change Log
 
+### Unreleased
+- Upgraded several dependencies
+- Upgraded to AGP 8.2.1
+- Upgraded to Kotlin 1.9.22
```

Comment:
Concern here - This would be a breaking change I'm pretty sure since bumping AGP and Kotlin would require the client to have the same Kotlin metadata (baked into AGP).  The app I'm working on currently is not Kotlin 1.9.22 ready so it would take some effort for us to pick this version of Harmony up.